### ISSUE TITLE
[White Label - Videocard]: "Translate audio" control present under video is not keyboard accessible.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5707]
### What is the PWD impact?

If the “Translate Audio” control under the video is not keyboard accessible, keyboard-only users are unable to access or activate the feature, preventing them from using the translation functionality and limiting equal access to content.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard navigation

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type Videocard word in type your message text box.
4.Navigate to the search results.
5. Navigate on "Translate audio" control present under video.
6.Observe that "Translate audio" is not keyboard accessible.

### What do you expect?

The “Translate Audio” control under the video should be fully keyboard accessible, allowing users to navigate to it and activate it using standard keyboard interactions (e.g., Tab and Enter/Space keys).

### What actually happened?

"Translate audio" control present under video is not keyboard accessible. Here we use tab, shift+tab and up/down arrow key.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/ebba4395-4538-4b2e-ba36-0d80528a3144

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/keyboard

### Adaptive Card JSON

```json

```
